tools/Utilities/IfeoTool/IfeoToolWindow.xaml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<windows:ThemeAwareWindow
+    x:Class="DevHome.IfeoTool.IfeoToolWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:windows="using:DevHome.Common.Windows"
+    xmlns:windowex="using:WinUIEx"
+    Width="640"
+    Height="480"
+    MinWidth="640"
+    MinHeight="480"
+    mc:Ignorable="d">
+    <Window.SystemBackdrop>
+        <MicaBackdrop />
+    </Window.SystemBackdrop>
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <StackPanel Orientation="Horizontal" Height="31">
+            <Image Source="/Images/dd.ico" Height="16" Width="16" Margin="8,2,0,0" VerticalAlignment="Center"/>
+            <TextBlock x:Name="IfeoToolTitle" 
+                    Style="{StaticResource CaptionTextBlockStyle}" 
+                    Margin="6,0,0,0" 
```

Comment:
>Margin="6,0,0,0" [](http://example.com/codeflow?start=20&length=16)

I'll beat Kristin to this... instead of using a margin here, use a "Spacing=6" in the StackPanel declaration instead.

Reply:
No, that would put 6,6,6,6 spacing around all elements. Spacing makes sense if you've got multiples of the same type of element, which isn't true here. In this case, we want different margins for the elements.